src/bool.cr

```diff
@@ -1,10 +1,14 @@
+require "./comparable"
```

Comment:
```suggestion
require "comparable"
```

The relative paths are used within each module. AFAIK
Using require "comparable" will use the regular lookup that allows among other things redefining the module via CRYSTAL_PATH. Not a main use case, but is better to not loose that.